jerry-core/ecma/base/ecma-alloc.h

```diff
@@ -30,108 +30,108 @@
  *
  * @return pointer to allocated memory
  */
-extern ecma_object_t *ecma_alloc_object (void);
+extern ecma_object_t * ecma_alloc_object (void);
```

Comment:
@ruben-ayrapetyan,

personally I don't like the following style:
```c
extern ecma_object_t * ecma_alloc_object (void);
```
a better one would be:
```c
extern ecma_object_t *ecma_alloc_object (void);
```
but that's just my preference. I think we should wait for @egavrin's opinion on the matter. 

Aside from this the patch looks good to me.